app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
+  before_save :downcase_email
+  validates :name,  presence: true, length: {maximum: Settings.users.max_name}
+  validates :email, presence: true,
+                    length: {maximum: Settings.users.max_email},
+                    format: {with: Settings.users.VALID_EMAIL_REGEX},
+                    uniqueness: true
+  has_secure_password
+  validates :password, presence: true,
+                      length: {minimum: Settings.users.min_password}
+  private
+  def downcase_email
+    email.downcase
```

Comment:
```suggestion
    email.downcase!
```
nếu k dùng ! thì email của e sẽ không dc downcase khi lưu vào DB đâu e

Reply:
em sửa rồi e merge luôn nhé ạ